torchdrivesim/traffic_controls.py

```diff
@@ -191,4 +286,23 @@ class StopSignControl(BaseTrafficControl):
     Stop sign, indicating the vehicle should stop and yield to cross traffic.
     Violations are not computed.
     """
-    pass
+    def compute_violation(self, agent_state: Tensor, time: int) -> Tensor:
```

Comment:
@lioutasb can you comment on the logic used to compute the stop sign violation here?

Reply:
Sorry I just saw the comment. I'll take a look.

Reply:
If I understand correctly, the idea here is to count a stop sign violation if the agent overlaps with the stop line offsetted 5 meters forward if it's within two timesteps from the moment it touches the original stop line. If this is the correct behavior then I think the code is fine.